docs/release-notes/8.16.asciidoc

```diff
@@ -1,6 +1,47 @@
 [[release-notes-header-8.16.0]]
 == 8.16
 
+[discrete]
+[[release-notes-8.16.6]]
+=== 8.16.6
+
+[discrete]
+[[known-issue-8.16.6]]
+==== Known issues
+
+// tag::known-issue[]
+[discrete]
+.Duplicate alerts can be produced from manually running threshold rules 
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+====
+// end::known-issue[]
+
+[discrete]
+[[bug-fixes-8.16.6]]
+==== Bug fixes
+* Fixes a bug that affected column widths in Timeline ({kibana-pull}214178[#214178]).
+* Fixes a bug that prevented you from being able to save event filters ({kibana-pull}213805[#213805]).
+* Allows Automatic Import to handle a variety of structures when parsing system logs ({kibana-pull}212611[#212611]).
+* Allows data view selections to persist when you save a Timeline ({kibana-pull}211343[#211343]).
+* Improves trusted applications performance by dropping Event Tracing for Windows (ETW) events earlier.
+* Improves ETW-TI performance by not enabling local executable virtual memory events for sandboxed and Antimalware-PPL processes.
+* Fixes the low probability of Tamper Protected {elastic-defend} upgrade failure.
```

Comment:
Idk if "tamper protected" should be cap? Please disregard if so. 
```suggestion
* Fixes the low probability of tamper-protected {elastic-defend} upgrade failure.
```

Reply:
According to these docs, it looks like it should be lower-case. Thanks, @jmikell821!

Reply:
Fixed in [0e5b413](https://github.com/elastic/security-docs/pull/6650/commits/0e5b413261aeb489eb360c169dd2aa23afb39457).